File: swift练习/Task 5/useUIView/useUIView/ViewController.m
//
//  ViewController.m
//  useUIView
//
//  Created by 张皓 on 4/20/16.
//  Copyright © 2016 张皓. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UIView *redView;
@property (weak, nonatomic) IBOutlet UIView *blueView;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    NSLog(@"redView frame = %@",NSStringFromCGRect(self.redView.frame));
    NSLog(@"redView frame = %@",NSStringFromCGRect(self.blueView.bounds));
    NSLog(@"redView center =%@",NSStringFromCGPoint(self.redView.center));

}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: swift练习/TASK 8/RecordSound/RecordSound/ViewController.swift
//
//  ViewController.swift
//  RecordSound
//
//  Created by 张皓 on 16/5/21.
//  Copyright © 2016年 张皓. All rights reserved.
//

import UIKit
import AVFoundation

class ViewController: UIViewController {
    
    var avRec: AVAudioRecorder!
    var audioFlieUrl: NSURL!
    var avPlayer: AVAudioPlayer!

    @IBAction func startRecordBtnClicked(sender: AnyObject) {
    }
    
    @IBAction func stopRecordBtnClicked(sender: AnyObject) {
    }
    
    @IBAction func playRecordBtnClicked(sender: AnyObject) {
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        audioFlieUrl = (NSFileManager.defaultManager().URLsForDirectory(NSSearchPathDirectory.DocumentDirectory, inDomains: NSSearchPathDomainMask.AllDomainsMask)[0] as NSURL).URLByAppendingPathComponent("rec")
        
//        avRec = try! AVAudioRecorder(URL: (audioFlie) 
        
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

